### PR TITLE
Update ar-EG.txt With Missing strings ((o_o))

### PR DESCRIPTION
**Pull request by Omaranwa**
If you are applying translations for an issue (or multiple), please list them below (if not, delete this text).

Applying for issue(s):
- #1367 
- #1386 
- #1390 
- #1399 
- #1400 
- #1411

### CODE REVIEW

**Comment by AaronVanGeffen** [comment]
No need to add untranslated English strings. The English language file will be used for those instead.

File: data/language/ar-EG.txt [hunk 1]
```diff
@@ -4948,22 +4974,392 @@ STR_DTLS    :
 #####################
 
 #RCT2
-[TOPSP1]
```
Comment (Gymnasiast) [inline data/language/ar-EG.txt]: This must be kept in.

File: data/language/ar-EG.txt [hunk 2]
```diff
@@ -4948,22 +4974,392 @@ STR_DTLS    :
...
+STR_NAME    :Condor Trains
 STR_DESC    :Riding in special harnesses below the track, riders experience the feeling of flight as they swoop through the air in Condor-shaped trains
-STR_CPTY    :4 راكبين لكل عربة
+STR_CPTY    :4 passengers per car
 
 #WW
 [CONGAEEL]
-STR_NAME    :Conger Eel Coaster
-STR_DESC    :A compact steel-tracked roller coaster where the Eel-shaped train travels through corkscrews and loops
-STR_CPTY    :4 راكبين لكل عربة
```
Comment (Gymnasiast) [inline data/language/ar-EG.txt]: STR_CPTY should be left in here.

**Comment by Omaranwa** [comment]
sorry i do not have time right now but i will do it later @AaronVanGeffen

**Comment by Omaranwa** [comment]
i just updated the files with the new strings from issues:
#1442 
#1432 
#1424 
#1419 
with fixing missing strings

File: data/language/ar-EG.txt [hunk 3]
```diff
@@ -4949,21 +4978,392 @@ STR_DTLS    :
 
 #RCT2
 [TOPSP1]
-STR_NAME    :اللفة العالية
-STR_DESC    :Passengers ride in a gondola suspended by large rotating arms, rotating forwards and backwards head-over-heels
-STR_CPTY    :8 راكبين
```
Comment (Gymnasiast) [inline data/language/ar-EG.txt]: As I said, do not remove STR_CPTY/STR_DESC entries, even if they are not present in `en-GB.txt`

File: data/language/ar-EG.txt [hunk 4]
```diff
@@ -4949,21 +4978,392 @@ STR_DTLS    :
...
+[SUBMAR]
+STR_NAME    :Submarines
 
 #WW
 [CONDORRD]
-STR_NAME    :ركوب الكندور
+STR_NAME    :Condor Trains
 STR_DESC    :Riding in special harnesses below the track, riders experience the feeling of flight as they swoop through the air in Condor-shaped trains
-STR_CPTY    :4 راكبين لكل عربة
+STR_CPTY    :4 passengers per car
```
Comment (Gymnasiast) [inline data/language/ar-EG.txt]: You changed Arabic back to English here.

File: data/language/ar-EG.txt (hunk 3 above)
Comment (Omaranwa) [reply data/language/ar-EG.txt]: oh i didn't see that!
i will do it
thanks

**Comment by Omaranwa** [comment]
check now

**Pull request merged**